src/lib.rs

```diff
@@ -219,6 +219,18 @@ unsafe impl<'a> Alloc for &'a Jemalloc {
     }
 }
 
+/// Return the usable size of the allocation pointed to by ptr.
+///
+/// The return value may be larger than the size that was requested during allocation.
+/// This function is not a mechanism for in-place `realloc()`;
+/// rather it is provided solely as a tool for introspection purposes.
+/// Any discrepancy between the requested allocation size
+/// and the size reported by this function should not be depended on,
+/// since such behavior is entirely implementation-dependent.
```

Comment:
Mind adding an `# Unsafety` section here as well?

Reply:
Done, though I couldn’t find info on undefined behavior with this function in http://jemalloc.net/jemalloc.3.html so I based it on what I recall hearing about allocators in general.